openquakeplatform/openquakeplatform/static/js/irv_viewer_PCP_primary.js

```diff
@@ -74,22 +74,27 @@ function Primary_PCP_Chart(projectDef, layerAttributes, selectedRegion) {
 
         // Get the data for each selected theme child
         var data = [];
-        // first setup an object with all regions and the plot element
+        // first setup an object with all regions and the plot element and 0 for each value
         var la = layerAttributes.features;
         for (var ia = 0; ia < selectedThemeChildren.length; ia++) {
             var temp = {};
             temp.plotElement = selectedThemeChildren[ia].field;
             for (var s = 0; s < la.length; s++) {
-                var bar = la[s].properties[selectedRegion];
-                temp[bar] = 0;
+                var eachReagion = la[s].properties[selectedRegion];
+                temp[eachReagion] = 0;
             }
             data.push(temp);
         }
 
+        // Poipulate the object created above with values
         for (var n = 0; n < data.length; n++) {
             for (var o = 0; o < layerAttributes.features.length; o++) {
                 var field = data[n].plotElement;
                 var value = layerAttributes.features[o].properties[field];
+
+                if (value === null) {
+                    value = 211.309;
```

Comment:
Whoops, thats not supposed to be there, it was just a test. But, yes, this gets addressed in PR #431